src/ssl/support.cc

```diff
@@ -390,9 +390,17 @@ ssl_verify_cb(int ok, X509_STORE_CTX * ctx)
 }
 
 void
-Ssl::SetupVerifyCallback(Security::ContextPointer &ctx)
+Ssl::SetupPeerVerification(Security::ContextPointer &ctx, bool verify, bool fail_if_no_peer_cert)
 {
-    SSL_CTX_set_verify(ctx.get(), SSL_VERIFY_PEER | SSL_VERIFY_FAIL_IF_NO_PEER_CERT, ssl_verify_cb);
+  int mode;
+
+  mode = SSL_VERIFY_NONE;
```

Comment:
No need for the C-style initialization and empty line above.
```suggestion
    int mode = SSL_VERIFY_NONE;
```

Please also indent the lines below properly: 4-spaces per level of indentation, and one space between "if" and their condition brackets.